data/generateDataCassandra.js

```diff
@@ -48,12 +48,14 @@ const restaurantItems = createRestaurantItemArray(restaurantItemNames.length);
 
 const createRestaurantItemsFile = (streamWriter, data) => {
   let i = -1;
+  let count = 0;
   const write = () => {
     let drained = true;
     do {
       i += 1;
+      count += 1;
       for (let j = 0; j < 15; j += 1) {
-        drained = streamWriter.write(`${data[i]},${restaurantItems[Math.floor(Math.random() * restaurantItems.length)]}\n`);
+        drained = streamWriter.write(`${count},${data[i]},${restaurantItems[Math.floor(Math.random() * restaurantItems.length)]}\n`);
```

Comment:
Does Cassandra need you to supply your own ID values here? I would assume that kind of thing would be built it, but I honestly don't know and now I'm just curious.

Reply:
Yeah, Cassandra doesn't have auto-incrementing IDs. Normally, I would use a UUID for Cassandra but since the code we inherited uses IDs to uniquely identify restaurants, I used IDs here. Ideally, I probably would've changed to UUIDs for Cassandra and just refactored the inherited code, but alas.. time. hahaha

Reply:
Ahh, ok, cools. Works for me.